doc/source/conf-postgres.rst

```diff
@@ -0,0 +1,57 @@
+.. _pmm.qan.postgres.conf:
+
+===============================================================================
+Configuring |postgresql| for Monitoring
+===============================================================================
+
+Monitoring |postgresql| metrics with the ``postgres_exporter`` can be turned on
+with ``pmm-admin add postgresql`` command. The ``postgresql`` alias will set up
+``postgresql:metrics`` and also ``linux:metrics`` on a host (for more
+information, see `Adding monitoring services <https://www.percona.com/doc/percona-monitoring-and-management/pmm-admin.html#pmm-admin-add>`_).
+
+Several parameters are expected by postgres_exporter to make things work. 
+
+The 1st parameter is an example URI, which will be built up based on the correct flags being passed to pmm-admin. An example of the URI is::
+
+   DATA_SOURCE_NAME=postgresql://postgres_exporter:password@172.17.0.2:5432/postgres?sslmode=disable
+
+``pmm-admin`` supports passing |postgresql| connection information via following flags:
+
+=============                 =================================================
+Flag                          Description 
+=============                 =================================================
+``--create-user``             create a new |postgresql| user
```

Comment:
Yes, they should work. You need to have privileged user e.g. `postgres` or you can create new one e.g. `root` the way I described here https://github.com/percona/pmm-client/pull/157

However since it still requires manual changes to PostgreSQL config file I might expect people will report this option not working as expected - I'm ok if we include it and get feedback on it, but I'm also ok if we decide to remove it.

Reply:
after speaking with @arvenil  it turns out that this change in config file is just switching to password authentication. May be we can place a not that password auth should be turned on to make --create-user work?